src/crypto/hsm/infineon/CHIPCryptoPALHsm_HKDF_trustm.cpp

```diff
@@ -1,59 +1,61 @@
 /*
-*
-* Copyright (c) 2021 Project CHIP Authors
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-* http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*/
+ *
+ * Copyright (c) 2021 Project CHIP Authors

```

Comment:
Careful about copyright dates that may be outdated.

Reply:
Hi @tcarmelveilleux Thanks for your comments. We already fixed this in the latest commit.